ansible/helm_external.yml

```diff
@@ -36,12 +36,12 @@
         server_type: cassandra
         network_interface: "{{ cassandra_network_interface }}"
 
-- hosts: "rmq-cluster"
-  become: false
-  tasks:
-    - name: Generate rabbitmq IPs for helm
-      include_tasks: tasks/helm_external.yml
-      vars:
-        external_dir_name: rabbitmq-external
-        server_type: rmq-cluster
-        network_interface: "{{ rabbitmq_network_interface }}"
+# - hosts: "rmq-cluster"
```

Comment:
I commenting out the best way to not execute this task? Wouldn't it be better to rely on some conditional (e.g. `when: rabbitmq_cluster.is_internal == false` (these value names likely don't exist yet, I made them up))?

Reply:
Created a tag for the rabbimq role and added --skip-tags and --tags to install as required.